.github/workflows/ci.yml

```diff
@@ -653,7 +673,7 @@ jobs:
   package:
     name: Package
     runs-on: ubuntu-latest
-    needs: [ lint-debug-test, docker-build-test, test-multi-os, test-with-svc ]
+    needs: [ lint-debug-test, test-multi-os, test-with-svc, test-aws-lambda ]
```

Comment:
before we package, we should have all tests run through.
Transitively, this still means that musl must have been build.
It does not mean that docker needs to have been build though.